pkg/controllers/dynakube/oneagent/daemonset/daemonset.go

```diff
@@ -380,3 +385,22 @@ func (dsInfo *builderInfo) getReadinessProbe() *corev1.Probe {
 
 	return defaultProbe
 }
+
+// isRootFsReadonly checks if the given version of the OneAgent supports the `ReadOnlyRootFilesystem` securityContext setting.
+// if the version is not set, ie.: unknown, we  consider the OneAgent to support `ReadOnlyRootFilesystem`.
+func (dsInfo *builderInfo) isRootFsReadonly() bool {
```

Comment:
I know it's not you, but based on convention it should be `b` or `bi`, dsInfo is a bit confusing. =)

Reply:
[this](https://github.com/Dynatrace/dynatrace-operator/pull/3183/commits/6b2b72a9db41dfcde17aeb5d39da74b78b8c1a49) is your fault now 😛

Reply:
love it!